packages/create-block/lib/templates/plugin/README.md.mustache

```diff
@@ -0,0 +1,114 @@
+
+#### src/editor.scss
+`editor.scss` is a file containing SCSS that styles the appearance of the block in the block editor. It requires a compile step which will result in `build/index.css`. 
+
+#### src/index.js
+`index.js` is the starting point for the JavaScript execution of the block. It imports the component exported by `edit.js`{{#isStaticVariant}} and the function exported by `save.js`{{/isStaticVariant}} and then executes `registerBlockType` passing as parameters the name of the block (retrieved from the `name` property in `block.json`) and an object containing an `edit` property{{#isStaticVariant}} and a `save` property{{/isStaticVariant}}.
+
+{{#isDynamicVariant}}
+#### src/render.php
+This file performs the server-side rendering in a **dynamic** block.
+{{/isDynamicVariant}}
+{{#isStaticVariant}}
+#### src/save.js
+In a **static** block `save.js` exports a function, `save()`, that determines the markup that will be saved to the `post_content` field in the `wp_posts` table when a post or page containing this block is saved, and hence determines how the block appears and functions in the front end. As with `edit.js` the function defined here receives props that will include the attributes defined in `block.json`.
+{{/isStaticVariant}}
+
+#### src/style.scss
+`style.scss` contains SCSS that styles the appearance of the block in the front end. It will also be used in the editor unless a style defined here is over-ridden by one in `editor.scss`. This file requires a compile step which will result in `build/style-index.css`. 
+
+#### src/view.js
+When this file is defined as the value of the `viewScript` property in `block.json` it will be enqueued on the front end of the site. You can use this file for JavaScript code that you want to run in the front-end on posts/pages that contain this block.
+
+### .editorconfig
+Unifies the coding style for different editors and IDEs
+
+### .gitignore
+Specifies files that should untracked and ignored by Git.
+
+### {{slugSnakeCase}}.php
+The main plugin file that contains the plugin's [header information](https://developer.wordpress.org/plugins/plugin-basics/header-requirements/) and initiates execution of the plugin.
+
+### package-lock.json
+This file is automatically generated for any operations where npm modifies either the `node_modules` tree, or `package.json`. It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates. [Learn more](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json)
+
+### package.json
+A manifest file for node that contains metadata about the project. Dependencies for the project and scripts are defined in this file. See [this guide](https://docs.npmjs.com/cli/v8/configuring-npm/package-json) to `package.json` to learn about the options defined here.
+
+### README.md
+[This file]
```

Comment:
Probably don't need this section at all. It's a bit self-evident :) Just my opinion, not strongly held though.